This patch causes the GCC to install the libstc++ into lib as opposed to
lib64 on X86_64 Linux targets. Usually the libstdc++ will be placed in lib64
during the install step. Because that's where 64bit libs go for linux targets
according to the FHS. If you don't like this but want to use linux as a target
you have to patch the toolchain which is what 64bithack.patch attempts to do.
One solution would be to have a distinctive OE target that doesn't support
multilib and has 64bit libs under lib. The advantage over the current
meta/recipes-devtools/gcc/gcc-4.6/64bithack.patch provided by oe-core is that
multilib upport is entirely disabled and MULTILIB_OSDIRNAMES is empty. This
prevents a broken toolchain in case --enable-multilib gets used.

Upstream-Status: Inappropriate

Signed-off-by: Ken Werner <ken.werner@linaro.org>

Index: gcc-linaro-4.6-2012.04/gcc/config.gcc
===================================================================
--- gcc-linaro-4.6-2012.04.orig/gcc/config.gcc
+++ gcc-linaro-4.6-2012.04/gcc/config.gcc
@@ -1310,7 +1310,7 @@ x86_64-*-linux* | x86_64-*-kfreebsd*-gnu
 		tm_file="${tm_file} knetbsd-gnu.h"
 		;;
 	esac
-	tmake_file="${tmake_file} i386/t-linux64 i386/t-crtstuff i386/t-crtpc i386/t-crtfm t-dfprules"
+	tmake_file="${tmake_file} i386/t-crtstuff i386/t-crtpc i386/t-crtfm t-dfprules"
 	;;
 i[34567]86-pc-msdosdjgpp*)
 	xm_file=i386/xm-djgpp.h
